ramlfications/utils.py

```diff
@@ -117,34 +135,23 @@ def save_data(output_file, mime_types):
     with open(output_file, "w") as f:
         json.dump(mime_types, f)
 
+def setup_logger(message):
+    log = logging.getLogger(__name__)
+    log.setLevel(logging.DEBUG)
+    console = logging.StreamHandler()
+    console.setLevel(logging.DEBUG)
+    msg = "{message} - %(levelname)s - %(message)s".format(message=message)
```

Comment:
Yeah I can see that it is confusing. 

The use of `%()` is a Python thing specifically for the `logging` module, see [docs](https://docs.python.org/2/library/logging.html#formatter-objects).  The `{}` paired with the `.format()` is the proper way for string replacement that is compatible with Python 2 and 3.

I assume that the `{message}` is @nvillalva 's way of customizing the message.